### DISCUSSION TITLE
Support for multiple github accounts

### DISCUSSION THREAD

**Discussion by terrisgit** [community/community#3969] [Mobile] [answered]
I would pay monthly for the ability to switch between accounts. Those who want to save money can continue changing their identities. I have MFA enabled so it's a pain.

**Comment by elisealix22** [comment, marked as answer]
Thank you so much for the request. I'm adding your note to our internal tracking issue! Please keep the feedback and/or bug reports coming as it is really helpful to us.

**Reply by getaaron** [reply]
Looks like it's supported for GitHub enterprise accounts, but it'd be nice to be able to add paid GitHub Team accounts as well (some companies require not linking personal account to the org).

**Reply by dzmpr** [reply]
Hi! Any updates?

**Comment by Wladefant** [comment]
Duplicate of #3337. Not good because that means the votes get divided up

**Comment by eliperkins** [comment]
Hey @terrisgit and folks, we've shipped functionality this week to allow you to log in with multiple accounts, like your personal and work accounts, at a the same time! [Read more about it in our Changelog post](https://github.blog/changelog/2023-08-15-log-in-with-multiple-github-accounts-on-github-mobile/).

<img width="1200" alt="image" src="https://github.com/community/community/assets/1051453/d2ba902e-2199-4625-948a-532d64a6f44e">

Download or update GitHub Mobile today from the [Apple App Store](https://apps.apple.com/us/app/github/id1477376905) or [Google Play Store](https://play.google.com/store/apps/details?id=com.github.android) to get started.

**Reply by qbedard** [reply]
Just noticed this yesterday! So happy to see it. Thank you! 😁😁😁
